easybuild/easyconfigs/g/Guile/Guile-3.0.7-GCCcore-11.2.0.eb

```diff
@@ -21,6 +21,7 @@ checksums = ['c7935b7a29e42443f6a35d35cf20ffa7d028c399303f872cd1219598a83656ae']
 builddependencies = [
     ('Autotools', '20210726'),
     ('binutils', '2.37'),
+    ('pkg-config', '0.29.2'),
```

Comment:
can we use `pkgconf` here instead?

Reply:
Yes, pkgconf is correct.

Reply:
pkg-conf was correct.

Reply:
I meant to use `pkgconf` as a drop-in replacement for `pkg-config`.
I went ahead and made the change for you if you don't mind.